HelpSource/Classes/BelaScopeUGen.schelp

```diff
@@ -8,10 +8,10 @@ This UGen only works on Bela
 ::
 This UGen is used internally to plot signals to Bela's Oscilloscope. Direct usage of this UGen should never be needed. See link::Classes/BelaScope::
 
+BelaScopeUGen allocates the internal buffer where link::Classes/BelaScopeOut:: UGens can write data to be scoped. Only one instance of BelaScopeUGen is allowed to be operational at the same time,
```

Comment:
maybe it's better phrased as "at the same time" -> "at any time" ?

Reply:
Sure, agree